crates/rattler/src/install/installer/mod.rs

```diff
@@ -303,14 +305,17 @@ impl Installer {
             )
         });
 
+        let prefix =
+            Prefix::create(prefix.as_ref().to_path_buf()).expect("failed to create prefix");
```

Comment:
Can you turn this into a proper error? Never unwrap unless you are certain this can never fail.